app/components/avo/index/resource_controls_component.rb

```diff
@@ -10,10 +10,7 @@ def initialize(resource: nil, reflection: nil, parent_model: nil, parent_resourc
   end
 
   def can_detach?
-    @reflection.present? &&
-      @resource.model.present? &&
-      is_has_many_association &&
-      authorize_association_for(:detach)
+    is_has_many_association ? super : false
```

Comment:
should we add a question mark at the end?
```suggestion
    is_has_many_association? ? super : false
```

Reply:
I'm divided. With the `?` on the end it is more clear and readable on a `if` condition: `if is_has_many_association?`... However if we use that method on a ternary condition it's not that readable because we end up with duplicated question mark. I can agree with both notations but in this case where we use ternary I prefer without the extra question mark